### PR TITLE
[docs] Add EAS Metadata docs

### PR DESCRIPTION
**Pull request by byCedric**
# Why

This is a draft to document the usage of EAS Metadata

# How

Added docs

# Test Plan

See docs: http://localhost:3002/eas-metadata/

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/eas-metadata/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import { Terminal } from '~/ui/components/Snippet';
+
+> ⚠️ EAS Metadata is in beta and subject to breaking changes.
+
+EAS Metadata helps you prepare your app for review in the app stores and helps you prevent common pitfalls that often could lead to an app rejection.
+
+## Prerequisites
+
+EAS Metadata is available starting from EAS CLI >= 0.54.0, and _only supports the App Store_.
```
Comment (EvanBacon) [inline docs/pages/eas-metadata/getting-started.md:14]:
```suggestion
EAS Metadata is available starting from EAS CLI >= 0.54.0, and _currently_ only supports the Apple App Store.
```

File: docs/pages/eas-metadata/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Create a local store configuration
+
+We first need to create our local **store.config.json** to get started. This file keeps track of all the information for the app stores. If you have an existing app in the stores, you can generate the configuration file by running:
```
Comment (EvanBacon) [inline docs/pages/eas-metadata/getting-started.md:18]:
```suggestion
To get started, create a **store.config.json** in the root project directory of your app. This file keeps track of all the information for the app stores. If you have an existing app in the stores, you can generate the configuration file by running:
```

File: docs/pages/eas-metadata/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+## Upload a new version of the app
+
+To create a new app version in the stores, you must upload a new binary. We can do this by running:
+
+<Terminal cmd={[
+  '# Build and deploy to the stores',
+  '$ eas build --auto-submit',
+  '# Or deploy a previous build to the stores',
+  '$ eas submit',
+]} />
```
Comment (brentvatne) [inline docs/pages/eas-metadata/getting-started.md:58]: ideally we can leave this terminal bit out and just link to some other docs page that tells people about getting their app on the store, or assume that they know how to do this already

File: docs/pages/eas-metadata/introduction.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+hideTOC: true
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+> ⚠️ EAS Metadata is in beta and subject to breaking changes.
+
+**EAS Metadata** is a service to help you provide all necessary information to the stores and get your app published.
+
+To get your app into the hands of your users, you have to publish the app on multiple app stores. During this process, you'll have to answer questions about complex topics that often don't apply to your app. After submitting the information, you can start the lengthy review process. If the reviewer finds any issues in the information you sent, you'll have to restart the review process, making the feedback loop worse.
```
Comment (brentvatne) [inline docs/pages/eas-metadata/introduction.md:13]:
```suggestion
To get your app into the hands of your users, you have to publish the app on multiple app stores. During this process, you'll have to answer questions about complex topics that often don't apply to your app. After submitting the information, you can start the lengthy review process. If the reviewer finds any issues in the information you sent, you'll have to restart the review process.
```

**Review by brentvatne — APPROVED** [review]

File: docs/pages/eas-metadata/getting-started.md (hunk 3 above)
Comment (brentvatne) [reply docs/pages/eas-metadata/getting-started.md:58]: testing a new tool

###### <a href="https://getunblocked.com"></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/5bf29d2c-23ae-4af8-8ccc-fe767cd2df79/thread/ef2c770e-87b7-4d63-8858-dc70e3c51f4a?message=611dfd65-7825-4d07-8daa-61f09335b64e).

Comment (byCedric) [reply docs/pages/eas-metadata/getting-started.md:58]: We can link to this if that makes more sense? https://docs.expo.dev/distribution/uploading-apps/

Comment (brentvatne) [reply docs/pages/eas-metadata/getting-started.md:58]: that works!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/5bf29d2c-23ae-4af8-8ccc-fe767cd2df79/thread/ef2c770e-87b7-4d63-8858-dc70e3c51f4a?message=42fca110-5457-4386-9424-5f204e7168c4).

Comment (byCedric) [reply docs/pages/eas-metadata/getting-started.md]: Done!

**Review by EvanBacon — APPROVED** [review]

File: docs/pages/eas-metadata/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+> ⚠️ EAS Metadata is in beta and subject to breaking changes.
+
+EAS Metadata helps you prepare your app for review by uploading most of the required app information using a simple JSON file. It also helps you prevent common pitfalls that may lead to a rejected app submission.
+
+## Prerequisites
+
+EAS Metadata is available starting from EAS CLI >= 0.54.0, and _currently_ only supports the Apple App Store.
+
+If you are using vscode, also make sure to [install the Expo plugin](https://github.com/expo/vscode-expo) for **store.config.json** auto-completion.
```
Comment (amandeepmittal) [inline docs/pages/eas-metadata/getting-started.md:16]:
```suggestion
If you are using VSCode, make sure to [install the Expo plugin](https://github.com/expo/vscode-expo) for **store.config.json** auto-completion.
```

Let's use the branding name for VSCode.

File: docs/pages/eas-metadata/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  }
+}
+```
+
+> EAS Metadata will use the **store.config.json** file name in your project root by default; you can change the name and location of the file by [configuring the `metadataPath` on the EAS Submit profile](../submit/eas-json.md#metadatapath).
+
+## Update the store config
+
+Now it's time to edit the **store.config.json** file and customize it to your app needs. A complete overview of what can be defined is coming soon. In the meantime, you can use the [vscode Expo plugin](https://github.com/expo/vscode-expo) and use the auto-completion.
```
Comment (amandeepmittal) [inline docs/pages/eas-metadata/getting-started.md:49]:
```suggestion
Now it's time to edit the **store.config.json** file and customize it to your app needs. A complete overview of what can be defined is coming soon. In the meantime, you can use the [VSCode Expo plugin](https://github.com/expo/vscode-expo) and use the auto-completion.
```

File: docs/pages/eas-metadata/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+<!-- Disabled due to missing config overview
+You can find all available options for the **store.config.json** in the [store configuration reference](./store-json.md). -->
+
+## Upload a new app version
+
+Before we can upload the **store.config.json**, you must upload a new binary of your app. You can [read more about uploading new binaries to stores here.](../distribution/uploading-apps.md).
```
Comment (amandeepmittal) [inline docs/pages/eas-metadata/getting-started.md:55]:
```suggestion
Before we can upload the **store.config.json**, you must upload a new binary of your app. You can [read more about uploading new binaries to stores here.](../distribution/uploading-apps.md).
```
```suggestion
Before you can upload the **store.config.json**, you must upload a new binary of your app. [Read more about uploading new binaries to stores](../distribution/uploading-apps.md).
```

As per our Docs Writing Style, let's stick to [writing in second person](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#write-in-the-second-person).

File: docs/pages/eas-metadata/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+After the binary is submitted and processed, we can upload the **store.config.json** to the app stores.
+
+## Upload the store config
+
+When you are happy with the **store.config.json** settings, we can start syncing that data to the app stores. All we need to do is one command:
```
Comment (amandeepmittal) [inline docs/pages/eas-metadata/getting-started.md:61]:
```suggestion
When you are happy with the **store.config.json** settings, start syncing that data to the app stores. All you need to do is run the following command:
```

**Review by amandeepmittal — COMMENTED** [review]
Made some small suggestions as per our Writing Style guide.

File: docs/pages/eas-metadata/getting-started.md (hunk 5 above)
Comment (byCedric) [reply docs/pages/eas-metadata/getting-started.md:16]: 👍

**Review by byCedric — COMMENTED** [review]
woops

**Review by amandeepmittal — APPROVED** [review]

**Review by Simek — APPROVED** [review]
Great work Cedric! 🙏

**Pull request merged**